### ISSUE TITLE
on the search results screen, the search button needs an aria-label

### ISSUE THREAD

**Issue by nickallan** [thorsten/phpMyFAQ#1129]
Specifically on the search results page, there is another search edit field with a search button. This search button doesn't have a text label, can you add aria-label="search" to this button.
